scripts/proteinInteractionEBI/README.md

```diff
@@ -0,0 +1,62 @@
+# Scripts for importing ontology dataset from the European Bioinformatics Institute (EMBL-EBI)
+
+This directory stores all scripts used to import datasets from the European Bioinformatics Institute (EMBL-EBI). 
+Here we only import the three subsets of the ontologies: "interaction detection method", "interaction type" and "database citation", which are commonly used in protein-protein interactions. 
+
+## Usage
+
+To generate 'BioOntologySchemaEnum.mcf' which contains all the schemas and "psimi2dcid.txt" which contains paired PSI-MI identifiers and DCID, run:
+
+```bash
+python3 parseEBI.py mi.owl 
+```
+
+If new reference sources which are not properties in dcs occur, 'BioEBINewSource.txt' containing such information will be generated as well.
+
+## Database format
+
+Data is available for download from
+http://psidev.info/groups/controlled-vocabularies. It can also be browsed at https://www.ebi.ac.uk/ols/ontologies/mi. 
+The ontologies dictionary has a tree structure. Note here that one parent node can have multiple child nodes and one child node can have multiple parent nodes as well.
+
+![Tree Structure](./graph/ontologyTree.png)
+
+![Multiple Parent Node](./graph/multipleParent.png)
+
+The original data file (mi.owl) uses "is_a" and "part_of" as the relation property to connect the child node to the parent node, however we don't distinguish these two and a property "specializationOf" is used for the child-parent connection.
+
+![Original Data Sample](./graph/originalDataSample.png)
+
+We make each term as an enumeration node of three subtrees with root nodes as: "database citation", "interaction detection method", and "interaction type". There are two main concerns that we didn't import all the nodes. First, we focus on the ontologies in protein-protein interaction and these three categories are the most commonly used. Secondly, importing too many general terms may cause confusion in our Data Commons knowledge graph. 
```

Comment:
I might reword as: "to avoid general terms from another ontology from polluting the Data Commons schema."